### ISSUE TITLE
ULS UNITS MANAGEMENT - Add alt text to Featured Resource images

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#465]
* Homepage - Featured Resource image is a link that does not have alternative text
